src/lib/containers/entity_finder/details/view/DetailsView.tsx

```diff
@@ -312,6 +319,28 @@ export const DetailsView: React.FunctionComponent<DetailsViewProps> = ({
     visibleTypes,
   ])
 
+  const NameRenderer = useCallback(
+    (props: EntityFinderTableCellRendererProps) => {
+      if (setCurrentContainer && isContainerType(props.rowData.entityType)) {
+        return (
+          <span
+            role="link"
+            className="EntityFinderTableCellContainerLink"
+            onClick={e => {
+              e.stopPropagation()
+              setCurrentContainer(props.rowData.id)
+            }}
+          >
+            {props.rowData.name}
+          </span>
+        )
+      } else {
+        return props.rowData.name
+      }
+    },
+    [setCurrentContainer],
+  )
```

Comment:
Renders the entity name, conditionally adds event to update the currentContainer if the entity is a container and the setter exists.